interface/forms/eye_mag/php/eye_mag_functions.php

```diff
@@ -5253,6 +5278,52 @@ function display_GlaucomaFlowSheet($pid, $bywhat = 'byday')
                 </tr>
                     <?php echo $gonios;
 
+                        $count = 0;
+                        $hideme = '';
+                    foreach ($priors as $visit) {
+                        if (($visit['ODKTHICKNESS'] > " ") || ($visit['OSKTHICKNESS'] > " ")) { // something is here
+                            if ($count > 0) {
+                                $hideme = "hideme_kthickness nodisplay";// show the first only, hide the rest for now
+                            }
+
+                            $pachymetry .= "<tr><td class='GFS_td_1 " . $hideme . "'>" . $visit['exam_date'] . "</td><td class='GFS_td " . $hideme . "' style='border:1pt dotted gray;'>" . $visit['ODKTHICKNESS'] . "</td><td class='GFS_td " . $hideme . "' style='border:1pt dotted gray;'>" . $visit['OSKTHICKNESS'] . "</td></tr>";
```

Comment:
escaping the $visit stuff with text() function:
```
$pachymetry .= "<tr><td class='GFS_td_1 " . $hideme . "'>" . text($visit['exam_date']) . "</td><td class='GFS_td " . $hideme . "' style='border:1pt dotted gray;'>" . text($visit['ODKTHICKNESS']) . "</td><td class='GFS_td " . $hideme . "' style='border:1pt dotted gray;'>" . text($visit['OSKTHICKNESS']) . "</td></tr>";
```

Reply:
nice spot @bradymiller , thanks